include/s3select_functions.h

```diff
@@ -1258,6 +1261,87 @@ struct _fn_like : public base_function
   }
 };
 
+struct _fn_likescape : public base_function
+{
+  void transform(std::string& s, size_t found)
```

Comment:
escape is "=" (or other escape char)
like is  "=_a=_" 
validate the result after transform

Reply:
Transform() gives ^_a.$. It should be ^_a_$

Reply:
> Transform() gives ^_a.$. It should be ^_a_$ .

what about the second _

Reply:
It should be there instead of '.'. Github comment not showing it.